models/cyberconnect/bnb/cyberconnect_bnb_events.sql

```diff
@@ -0,0 +1,150 @@
+    {% if not is_incremental() %}
+    AND a.evt_block_time >= '{{project_start_date}}'
+    {% endif %}
+
+),
+    
+cyberconnect_essence_register as (
+    select 
+            'bnb' as blockchain,
+            'cyberconnect' project,
+            'create' action,
+            'RegisterEssence' name,
+            a.contract_address,
+            a.evt_block_time,
+            date_trunc('day', a.evt_block_time) AS block_date,
+            a.evt_block_number,
+            'Mint' evt_type,
+            a.evt_tx_hash,
+            b.to buyer,
+            '0x0000000000000000000000000000000000000000' as seller,
+            b.handle,
+            a.profileId profile_id,
+            a.essenceId content_id,
+            a.essenceTokenURI content_uri
+    from {{source('link3_profile_bnb', 'ProfileNFT_evt_RegisterEssence')}} a
+    left join {{source('link3_profile_bnb', 'ProfileNFT_evt_CreateProfile')}} b on a.profileId = b.profileId
+    WHERE 1=1
+    {% if is_incremental() %}
+    AND a.evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+    {% if not is_incremental() %}
+    AND a.evt_block_time >= '{{project_start_date}}'
+    {% endif %}
+),
+
+-- essence collect by profile or address
+cyberconnect_essence_collect as (
+    select 
+            'bnb' as blockchain,
+            'cyberconnect' project,
+            'collect' action,
+            'CollectEssence' name,
+            a.contract_address,
+            a.evt_block_time,
+            date_trunc('day', a.evt_block_time) AS block_date,
+            a.evt_block_number,
+            a.evt_tx_hash,
+            'Mint' evt_type,
+            a.collector buyer,
+            c.to seller,
+            c.handle,
+            a.profileId profile_id,
+            a.essenceId content_id,
+            b.essenceTokenURI content_uri
+    from {{source('link3_profile_bnb', 'ProfileNFT_evt_CollectEssence')}} a
+    left join {{source('link3_profile_bnb', 'ProfileNFT_evt_RegisterEssence')}} b on a.essenceId = b.essenceId and a.profileId = b.profileId
+    left join {{source('link3_profile_bnb', 'ProfileNFT_evt_CreateProfile')}} c on a.profileId = c.profileId
+    WHERE 1=1
+    {% if is_incremental() %}
+    AND a.evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+    {% if not is_incremental() %}
+    AND a.evt_block_time >= '{{project_start_date}}'
+    {% endif %}
+),
+
+cyberconnect_subscribe as (
+    select 
+            'bnb' as blockchain,
+            'cyberconnect' project,
+            'collect' action,
+            'Subscribe' name,
+            a.contract_address,
+            a.evt_block_time,
+            date_trunc('day', a.evt_block_time) AS block_date,
+            a.evt_block_number,
+            a.evt_tx_hash,
+            'Mint' evt_type,
+            a.sender buyer,
+            b.to seller,
+            b.handle,
+            a.profile_id,
+            NULL AS  content_id,
+            NULL AS  content_uri
```

Comment:
Hi @nazihkalo, do you know the specific type for the NULL columns?
Maybe we can cast null to the expected types (maybe '')